tests/plugins/autolink/manual/autolink.html

```diff
@@ -24,5 +24,13 @@ <h3>URLs that shouldn't be turned into links</h3>
 </textarea>
 
 <script>
-	CKEDITOR.replace( 'editor1' );
+	CKEDITOR.replace( 'editor1', {
+		on: {
+			pluginsLoaded: function() {
```

Comment:
Sure it may save a couple of lines of code duplication. In your proposal, you will have a problem with config customization if you decide to change `on` property. It looks cleaner and without mentioned limitation, if the whole editor is passed as an argument.

Reply:
After talking it makes more sense to put this helper in #2693 PR. See https://github.com/ckeditor/ckeditor-dev/pull/2693#issuecomment-463967760